marathon_lb.py

```diff
@@ -1574,12 +1634,12 @@ def regenerate_config(marathon, config_file, groups, bind_http_https,
                       ssl_certs, templater, haproxy_map):
     domain_map_array = []
     app_map_array = []
-    apps = get_apps(marathon)
+    apps, apps_id_label_list = get_apps(marathon, [], groups)
```

Comment:
If function `get_apps` is defined as `get_apps(marathon, apps=[], groups=None):`, I think we can refer to this function as follows:

`apps, apps_id_label_list = get_apps(marathon, groups=groups)`